### PR TITLE
Accueil page

### PR DESCRIPTION
**Pull request by ozen2**

### CODE REVIEW

File: client/src/assets/images/ImageFondFixe.jpg
Comment (pjroggy) [inline client/src/assets/images/ImageFondFixe.jpg:1]: a supprimer

File: client/src/assets/images/ImageFondMobil.png
Comment (pjroggy) [inline client/src/assets/images/ImageFondMobil.png:1]: a supprimer

File: client/src/assets/images/Rectangle-44.svg
Comment (pjroggy) [inline client/src/assets/images/Rectangle-44.svg:1]: a supprimer

File: client/src/assets/images/TerreMobil.jpg
Comment (pjroggy) [inline client/src/assets/images/TerreMobil.jpg:1]: a supprimer

File: client/src/components/Slogan.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import "../styles/Slogan.scss";
+
+function Slogan() {
+  return (
+    <section className="slogan">
+      <h4>Votre voyage</h4>
+      <h3>INTERSTELLAIRE</h3>
+      <h4>commence</h4>
+      <h3>ICI</h3>
+    </section>
```
Comment (pjroggy) [inline client/src/components/Slogan.jsx]: section or hgroup?

File: client/src/styles/Footer.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    a {
+      img {
+        width: 1.5rem;
+      }
+    }
+  }
+
+  p {
+    font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/Footer.scss]: use $bodyFont: "Varela Round", sans-serif;

File: client/src/styles/Footer.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    font-size: 0.3rem;
+    opacity: 50%;
+    text-align: center;
+    color: white;
```
Comment (pjroggy) [inline client/src/styles/Footer.scss]: use $whiteColor: #fff8f0;

File: client/src/styles/Header.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  flex-direction: column;
+  text-align: center;
+  color: white;
+  background-image: url(/src/assets/images/carlos-kenobi-Knla5kYvsRE-unsplash.jpg);
+  background-size: cover;
+  height: 10rem;
+  background-repeat: no-repeat;
+
+  h2 {
+    font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/Header.scss]: use $headerFont: "Tomorrow", sans-serif;

File: client/src/styles/Header.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    margin-top: 2rem;
+    text-shadow: 0 0 15px black;
+  }
+  h1 {
+    font-family: Tomorrow;
```
Comment (pjroggy) [inline client/src/styles/Header.scss]: use $headerFont: "Tomorrow", sans-serif;

File: client/src/styles/Header.scss [hunk 6]
```diff
@@ -0,0 +1,52 @@
+.titleSlider {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  color: white;
```
Comment (pjroggy) [inline client/src/styles/Header.scss]: use $whiteColor: #fff8f0;

File: client/src/styles/HomeCards.scss [hunk 7]
```diff
@@ -0,0 +1,108 @@
+.sectionTitle {
+  font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/HomeCards.scss]: use $bodyFont: "Varela Round", sans-serif;

File: client/src/styles/HomeCards.scss [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+      flex-direction: row;
+      font-family: Varela Round;
+      font-size: 0.8rem;
+      gap: 4rem;
+      margin-top: 0.5rem;
+    }
+
+    a {
+      text-decoration: none;
+      color: white;
```
Comment (pjroggy) [inline client/src/styles/HomeCards.scss]: use $whiteColor: #fff8f0;

File: client/src/styles/HomeCards.scss [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+      background-color: #a05cf9;
+      width: 7rem;
+      text-align: center;
+      font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/HomeCards.scss]: use $bodyFont: "Varela Round", sans-serif;

File: client/src/styles/HomeCards.scss [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+
+    a:hover {
+      background-color: #5907c4;
+    }
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .sectionTitle {
+    font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/HomeCards.scss]: use $bodyFont: "Varela Round", sans-serif;

File: client/src/styles/Links.scss [hunk 11]
```diff
@@ -7,8 +7,8 @@
     font-family: Varela Round;
```
Comment (pjroggy) [inline client/src/styles/Links.scss]: use $bodyFont: "Varela Round", sans-serif;

File: client/src/styles/app.scss [hunk 12]
```diff
@@ -1,5 +1,47 @@
+@import url("https://fonts.googleapis.com/css2?family=Tomorrow:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Varela+Round&display=swap");
+
+$whiteColor: #fff8f0;
+$blackColor: #011427;
+$headerFont: "Tomorrow", sans-serif;
+$bodyFont: "Varela Round", sans-serif;
+
 * {
```
Comment (pjroggy) [inline client/src/styles/app.scss:8]: may be can we use html instead of *

File: client/src/components/Header.jsx [hunk 13]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+import Links from "./Links";
+import "../styles/Header.scss";
+
+function Header({ planetName }) {
+  return (
+    <hgroup className="titleSlider">
```
Comment (Yoann-Fortin) [inline client/src/components/Header.jsx]: Wrong use with this tag

File: client/src/components/HomeCards.jsx [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+    <>
+      <h2 className="sectionTitle">{sectionTitle}</h2>
+      <section className="allCards">
+        <section className="Cards">
+          <figcaption>
+            <img id="left" src={imageLeft} alt="" />
+            <img id="mid" src={imageMid} alt="" />
+            <img id="right" src={imageRight} alt="" />
+          </figcaption>
+          <hgroup>
```
Comment (Yoann-Fortin) [inline client/src/components/HomeCards.jsx:24]: Wrong uses to this html tag

File: client/src/components/HomeCards.jsx [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+  planetTitleMid,
+  planetTitleRight,
+  imageLeft,
+  imageMid,
+  imageRight,
+}) {
+  return (
+    <>
+      <h2 className="sectionTitle">{sectionTitle}</h2>
+      <section className="allCards">
```
Comment (Yoann-Fortin) [inline client/src/components/HomeCards.jsx]: Why section in section tag without title ?

File: client/src/components/Slogan.jsx [hunk 16]
```diff
@@ -0,0 +1,13 @@
+import "../styles/Slogan.scss";
+
+function Slogan() {
+  return (
+    <section className="slogan">
```
Comment (Yoann-Fortin) [inline client/src/components/Slogan.jsx]: Wrong uses to this tags

File: client/src/pages/Home.jsx [hunk 17]
```diff
@@ -0,0 +1,39 @@
...
+import HomeCards from "../components/HomeCards";
+import Header from "../components/Header";
+import Slogan from "../components/Slogan";
+import "../styles/app.scss";
+
+function Home() {
+  return (
+    <>
+      <header>
+        <Header planetName="Neptune" />
```
Comment (Yoann-Fortin) [inline client/src/pages/Home.jsx]: The component name isn't ideal, or you'd have to use the header tag in the component.

File: client/src/pages/Home.jsx
Comment (Yoann-Fortin) [inline client/src/pages/Home.jsx:1]: Why use header anf footer component here and not in your App component ?

File: client/src/styles/Footer.scss [hunk 18]
```diff
@@ -0,0 +1,32 @@
+@media screen and (min-width: 320px) {
```
Comment (Yoann-Fortin) [inline client/src/styles/Footer.scss]: Wrong use of this media query

File: client/src/styles/Header.scss (hunk 6 above)
Comment (Yoann-Fortin) [reply client/src/styles/Header.scss]: exactly

File: client/src/components/HomeCards.jsx [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+        <section className="Cards">
+          <figcaption>
+            <img id="left" src={imageLeft} alt="" />
```
Comment (Yoann-Fortin) [inline client/src/components/HomeCards.jsx:20]: Why use id here ?

File: client/src/styles/app.scss [hunk 20]
```diff
@@ -1,5 +1,47 @@
...
+a,
+button {
+  font-family: $bodyFont;
+  color: $whiteColor;
+}
+
+@media screen and (min-width: 992px) {
+  body {
+    background-image: url("/src/assets/images/ImageFondDesk.png");
+    background-size: cover;
```
Comment (Yoann-Fortin) [inline client/src/styles/app.scss]: duplicate code

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**